README.md

```diff
@@ -2,17 +2,34 @@
 
 ⚠️ This project is currently under active development. Things might break. Feel free to check the open issues & create new ones.
 
-*The best way to get started building decentralized applications on Ethereum!*
-
-A new version of [scaffold-eth](https://github.com/scaffold-eth/scaffold-eth/tree/master) with its core functionality. Built using NextJS, RainbowKit, Wagmi and Typescript.
+Scaffold-Eth 2 is a toolkit for building decentralized applications (dapps) on the Ethereum blockchain. It's a new version of [scaffold-eth](https://github.com/scaffold-eth/scaffold-eth/tree/master) with its core functionality. Built using NextJS, RainbowKit, Wagmi and Typescript.
 
 - ✅ Contract component to easily edit the smart contracts and view & test the contract on your frontend
 - 🔥 Burner wallet & local faucet
 - 🔐 Integration with the different wallet providers
 
----
+# Contents
```

Comment:
(Web2 SEO Carlos kicking in)

Can we increase the heading indenting by one (##) in each heading (except the title Scaffold-Eth 2)??

"#" => renders as h1 and it should be only one in the whole README.
"##" h2 / "###" h3, etc